### ISSUE TITLE
DropboxClientsManager.setupWithAppKey crashes on release build.

### ISSUE THREAD

**Issue by hiroyachiba** [dropbox/SwiftyDropbox#131]
DropboxClientsManager.setupWithAppKey crashes on release build. (SwiftyDropbox 4.1.0, iOS 10.1.1, xcode 8.1)
It does not crash on debug build.
It does crash on real device, it doesn't crash on simulator.
The crash didn't happen before.  I didn't change the code, but I updated iOS and xcode.

<img width="993" alt="stacktrace" src="https://cloud.githubusercontent.com/assets/6658346/20038102/09601498-a471-11e6-9350-645ad5525f18.png">

**Comment by smarx** [comment]
See also https://github.com/Alamofire/Alamofire/issues/1741.

**Comment by scobbe** [comment]
@hiroyachiba: It looks like this failure is happening at the Alamofire level. I was not able to reproduce it (SwiftyDropbox 4.1.0, iOS 10.1.1, xcode 8.1).

However, I did update SwiftyDropbox to use Alamofire 4.0.1. Please pull the latest version of SwiftyDropbox (v4.1.1) and see if the problem still remains.

Thanks for your patience!

**Comment by hiroyachiba** [comment]
@smarx @scobbe Thank you for your reply.
I just tried SwiftyDropbox v4.1.1 and the problem still does happen.
I've already been using Alamofire 4.0.1, so no change in my environment. 
BTW, the problem happens only on real device (not simulator), release build.  The problem doesn't happen on other conditions.
If it's an Alamofire issue, we may have to wait for them...

**Comment by scobbe** [comment]
@hiroyachiba: Thanks for your reply. I know you attached the stack trace, but what is the actual error that you receive during the crash?

**Comment by hiroyachiba** [comment]
The error I get is, the program crashes with
Thread 1: EXC_BAD_ACCESS (code=1, address=0x20)
at
let queue = DispatchQueue(label: "org.alamofire.session-manager." + UUID().uuidString)
(SessionManager.swift)

**Comment by hiroyachiba** [comment]
Close this bug since this may be xcode issue.
In https://github.com/Alamofire/Alamofire/issues/1741, one developer reported,
"i don't know why but when copy workspace into another folder and work form this place everything is fine. i dont have same exception. but when try to work in from old folder it has exception ".
I did the same thing, copy the project to other folder, and the crash does not happen anymore.
I changed the folder name and the crash does not happen.  I change the folder name back to the original and the crash does happen.
